wordpress-mobile/WordPress-Android
# [JP Install Full Plugin] Dashboard / Menu card

Fixes #17836

Work in progress. A few things are still needed to finish this task:
- [x] Start the install flow for the Full Plugin (which is currently WIP)
- [x] Implement the content text logic
- [ ] Unit tests (will be handled in another PR)

To test:
- Install app and select site with individual plugin(s);
- Open home screen: the JP full plugin install card should be shown with `"Learn more"` in the card and `"Hide this"` in the overflow menu;
- When `"Learn more"` is tapped, the JP plugin install onboarding should be shown;
- When `"Hide this"` is tapped, the card should disappear and never appear again (unless app is reinstalled).

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
